src/submission/migrations/0041_auto_20200423_1338.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-04-23 12:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import submission.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submission', '0040_article_projected_issue'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='article',
+            name='primary_issue',
+            field=models.ForeignKey(blank=True, help_text='You can only assign an Issue that this Article is part of.You can add this article to and Issue from the Issue Manager.', null=True, on_delete=django.db.models.deletion.SET_NULL, to='journal.Issue'),
+        ),
+        migrations.AlterField(
+            model_name='article',
+            name='stage',
+            field=submission.models.DynamicChoiceField(blank=True, choices=[('Unsubmitted', 'Unsubmitted'), ('Unassigned', 'Unassigned'), ('Assigned', 'Assigned to Editor'), ('Under Review', 'Peer Review'), ('Under Revision', 'Revision'), ('Rejected', 'Rejected'), ('Accepted', 'Accepted'), ('Editor Copyediting', 'Editor Copyediting'), ('Author Copyediting', 'Author Copyediting'), ('Final Copyediting', 'Final Copyediting'), ('Typesetting', 'Typesetting'), ('Proofing', 'Proofing'), ('pre_publication', 'Pre Publication'), ('Published', 'Published'), ('preprint_review', 'Preprint Review'), ('preprint_published', 'Preprint Published')], default='Unsubmitted', max_length=200),
+        ),
```

Comment:
@ajrbyers Why is this change being detected?

Reply:
Its reverting after the Typesetting Plugin went in I think.

Reply:
I've added it on a commit of its own, mind regenerating this one?